<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapAccountWizard</class>
 <widget class="QWizard" name="MapAccountWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Mapping</string>
  </property>
  <widget class="QWizardPage" name="pageBackends">
   <property name="title">
    <string>Select Backend</string>
   </property>
   <property name="subTitle">
    <string>Please select a backend from the list below...</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="backendsList">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Module</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addBackendButton">
        <property name="text">
         <string>Select a backend</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="pageAccounts">
   <property name="title">
    <string>Select Account</string>
   </property>
   <property name="subTitle">
    <string>Select what &lt;b&gt;online&lt;/b&gt; account you want to map to the &lt;b&gt;application&lt;/b&gt; account...</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTreeWidget" name="accountsList">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Balance</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
